### PR TITLE
Fix problems detected by @phpstan

### PR DESCRIPTION
**Pull request by szepeviktor**
`phpstan analyze src/ -l 5`

@assertchris

### CODE REVIEW

File: src/Manager/GroupProcessManager.php [hunk 1]
```diff
@@ -81,7 +81,7 @@ public function tick()
             return false;
         }
 
-        $this->queuing = array_shift($this->waiting);
+        $this->queuing = [array_shift($this->waiting)];
```
Comment (szepeviktor) [inline src/Manager/GroupProcessManager.php:84]: Or is it `$this->queuing[] = array_shift($this->waiting);` ??

**Comment by szepeviktor** [comment]
@assertchris Closed without reaction?

File: src/Shell/BashShell.php [hunk 2]
```diff
@@ -12,7 +12,7 @@ final class BashShell implements Shell
      * @param string $format
      * @param array $parameters
      *
-     * @return array
+     * @return string
```
Comment (szepeviktor) [inline src/Shell/BashShell.php:15]: This is indeed a mistake. `array` is the correct type
https://www.php.net/manual/en/function.exec.php
Fixed the interface's type in https://github.com/asyncphp/doorman/pull/34

File: src/Manager/GroupProcessManager.php (hunk 1 above)
Comment (szepeviktor) [reply src/Manager/GroupProcessManager.php:84]: Ooops!
The same type as `$waiting` :)

**Comment by assertchris** [comment]
Sorry about the hassle - I change the default branch and it closed this PR automatically. Would you like me to apply the change directly, or would you like to re-crate/rebase the PR?

**Comment by szepeviktor** [comment]
Yes please.

**Pull request closed without merging**